### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by MichaelAAbbott**

### CODE REVIEW

File: app/views/SubmissionsView.scala.html [hunk 1]
```diff
@@ -89,6 +100,61 @@ <h2 class="govuk-heading-s">
...
+              </a>
+            </li>
+          }
+          @if(currentPage > 3) {
+            <li class="govuk-pagination__item govuk-pagination__item--ellipses">&ctdot;</li>
+          }
+
+          @for(i <- (currentPage - 1).max(1) to (currentPage + 1).min(totalCount / limit)) {
+            <li class="govuk-pagination__item @if(i == currentPage) {govuk-pagination__item--current}">
+              <a class="govuk-link govuk-pagination__link" href="@routes.SubmissionsController.onPageLoad(service, status, created, Some((i - 1) * limit))" aria-label="Page 7" aria-current="page">
```
Comment (TheNickWilson) [inline app/views/SubmissionsView.scala.html]: Should the page number in the aria-label here be dynamic?

File: app/views/SubmissionsView.scala.html [hunk 2]
```diff
@@ -89,6 +100,61 @@ <h2 class="govuk-heading-s">
...
+            </li>
+          }
+
+          @if(currentPage < lastPage - 2) {
+            <li class="govuk-pagination__item govuk-pagination__item--ellipses">&ctdot;</li>
+          }
+
+          @if(currentPage < lastPage - 1) {
+            <li class="govuk-pagination__item">
+              <a class="govuk-link govuk-pagination__link" href="@routes.SubmissionsController.onPageLoad(service, status, created, Some(totalCount - limit))" aria-label="Page 42">
```
Comment (TheNickWilson) [inline app/views/SubmissionsView.scala.html]: Should the page number in the aria-label here be dynamic?

**Review by TheNickWilson — CHANGES_REQUESTED** [review]

File: app/views/SubmissionsView.scala.html (hunk 1 above)
Comment (MichaelAAbbott) [reply app/views/SubmissionsView.scala.html]: [Sorted with latest commit :)](https://github.com/hmrc/dms-submission-admin-frontend/pull/7/commits/16cc18066af12280b59eee5cfdbcf3426bc31522)

File: app/views/SubmissionsView.scala.html (hunk 2 above)
Comment (MichaelAAbbott) [reply app/views/SubmissionsView.scala.html]: [Sorted with latest commit :)](https://github.com/hmrc/dms-submission-admin-frontend/pull/7/commits/16cc18066af12280b59eee5cfdbcf3426bc31522)

**Review by TheNickWilson — APPROVED** [review]

**Pull request merged**